clouddriver-appengine/clouddriver-appengine.gradle

```diff
@@ -22,9 +22,9 @@ dependencies {
   implementation "io.spinnaker.kork:kork-config"
   implementation "io.spinnaker.kork:kork-cloud-config-server"
   implementation "io.spinnaker.kork:kork-moniker"
+  implementation "io.spinnaker.kork:kork-retrofit"
   implementation "com.netflix.spectator:spectator-api"
   implementation "com.squareup.okhttp:okhttp"
-  implementation "com.squareup.retrofit:retrofit"
```

Comment:
Sorry if I asked you to remove this.  It needs to stay for e.g. [RetrofitAdapter](https://github.com/spinnaker/clouddriver/blob/6240ce89ff1564787c9391804db3131d92035e12/clouddriver-appengine/src/main/java/com/netflix/spinnaker/clouddriver/appengine/config/AppengineConfigurationProperties.java#L30C2-L30C29).